lib/src/tools/selenium.dart

```diff
@@ -0,0 +1,157 @@
+// Copyright 2015 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+library dart_dev.src.tools.selenium;
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:uuid/uuid.dart';
+
+import 'package:dart_dev/src/task_process.dart';
+import 'package:dart_dev/src/util.dart';
+
+/// An installer and runner for a standalone Selenium server.
```

Comment:
I think I originally intended for this Helper to install selenium, but that's no longer the case. Should remove "installer" from this comment.